integration/base/src/main/java/com/inrupt/client/integration/base/AuthenticationScenarios.java

```diff
@@ -159,31 +163,28 @@ static void setup() {
 
         privateContainerURI = URIBuilder.newBuilder(URI.create(podUrl))
                 .path(PRIVATE_RESOURCE_PATH + "/").build();
+        //if a tests fails it can be that the cleanup was not properly done, so we do it here too
```

Comment:
If _tests_ fail, the cleanup (`@AfterAll`) gets run, it's only if the setup (`@BeforeAll`) fails that the cleanup doesn't happen.